django_gcloud_storage/__init__.py

```diff
@@ -162,9 +163,14 @@ def _save(self, name, content):
 
         # Required for InMemoryUploadedFile objects, as they have no fileno
         total_bytes = None if not hasattr(content, 'size') else content.size
+        content_type = None if not hasattr(content, 'content_type') else content.content_type
+        if content_type is None:
+            _, file_extension = os.path.splitext(name)
+            content_type = mimetypes.types_map.get(file_extension, None)
```

Comment:
This code could be simplified, take a look how django-storages handles it: https://github.com/jschneier/django-storages/blob/f009463/storages/backends/s3boto3.py#L475

I'd prefer:

```python
if content_type is None:
    content_type, _ = mimetypes.guess_type(name)
```